utils/SwiftIntTypes.py

```diff
@@ -14,6 +14,13 @@
 # Number of bits in the biggest int type
 int_max_bits = max(_all_integer_type_bitwidths)
 
+def int_max(bits, signed): 
+    bits = bits - 1 if signed else bits 
+    bits = max(bits, 0)
+    return (1 << bits) - 1
+    
+def int_min(bits, signed):
+    return -1 * int_max(bits, signed) - 1 if signed else 0
```

Comment:
To add some clarity, could you add parens around the first part of the expression? Otherwise it is not quite clear whether it's:
```python
(-1 * int_max(bits, signed) - 1) if signed else 0
```
or
```python
-1 * int_max(bits, signed) - (1 if signed else 0)
```